src/components/stack/BlockItem.js

```diff
@@ -12,21 +12,180 @@ class BlockItem extends Component {
 
     this.state = {
       hover: false,
-      modTitle: null,
-      modDescription: null,
-      modDurationWork: null,
-      modDurationBreak: null,
+      modTitle: null, //title of the task block
+      modDescription: null, //description of the task block
+      modDurationWork: null, //time length in seconds of burst
+      modDurationBreak: null, //time length in seconds of break
       focusState: FOCUS_NONE,
     };
+
+    this.handleOnHover = this.handleOnHover.bind(this);
+    this.handleOnNotHover = this.handleOnNotHover.bind(this);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.handleCancel = this.handleCancel.bind(this);
+    this.handleEdit = this.handleEdit.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+  }
+
+  //onMassEnter and onMassLeave
+  //onhover, check what state it's in, based on that change the local state
+
+  handleOnHover = () => {
```

Comment:
Prefer functions over lambdas for class methods. Personal choice but I just think it's weird to have lambdas as class methods lol.
Prefer 'myFunc(args) {}' over 'myFunc = (args) => {}' for class methods.

Reply:
Okay sounds good. I don't really know when you're supposed to use which, I just prefer lambdas but I don't really mind lol. Okay got it sir 👍